Vwitter/VentCell.m

```diff
@@ -0,0 +1,68 @@
+//
+//  VentCell.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/14/22.
+//
+
+#import "VentCell.h"
+
+#import "VWHelpers.h"
+#import "VWUser.h"
+#import "DateTools.h"
+
+@implementation VentCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    // Initialization code
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+
+    // Configure the view for the selected state
+}
```

Comment:
nit: Same here, these can be removed since they only just call the superclass implementation of the method, which is the default behavior when a subclass doesn't implement (override) a method that a super class owns